homeassistant/components/bluesound/media_player.py

```diff
@@ -1,1009 +0,0 @@
-"""Support for Bluesound devices."""
```

Comment:
Please don't delete this file.

Reply:
I am still on a learning curve with git, did some reading up but am still struggling how to remove this file from my pull request while not deleting it... do you have any suggestions on how to best accomplish this?

Reply:
I would do an interactive rebase and delete the last commit, then force push.

Reply:
Done the rebase... still not clear on how to remove that file from this branch. I found out that for some reason it has been there from the beginning. Might be best to just stash the changes, discard this branch and start a new branch and PR?

Reply:
Looks like it's the three first commits in this branch that changes the bluesound integration. If those commits are removed the branch should be clean.

If you prefer to start over in a new branch and PR, that's ok too.

Reply:
That was it I think; just dropped those three commits in interactive rebase, now all seems good. Thanks for helping me out with this one..!